LEAF_Request_Portal/templates/print_form.tpl

```diff
@@ -98,6 +99,7 @@ function doSubmit(recordID) {
 		data: {CSRFToken: '<!--{$CSRFToken}-->'},
 		success: function(response) {
             if(response.errors.length == 0) {
+                $('.submitStatus').text('Request submmited');
```

Comment:
This should use an ID selector instead of a class selector.

Reply:
Done